test/fixtures/lookup_keys.yml

```diff
@@ -60,29 +60,3 @@ seven:
   path: "organization,location\nhostgroup"
   override: true
   type: PuppetclassLookupKey
-
-two:
-  key: ssl_port
-  key_type: string
-  validator_type: list
-  validator_rule: '443,8443'
-  puppetclass: one
-  default_value: 443
-  type: VariableLookupKey
-
-four:
-  key: special_info
-  key_type: string
-  validator_rule:
-  default_value: 'secret'
-  override: true
-  puppetclass: two
-  path: "organization,location\nxyz\nhostgroup"
-  type: VariableLookupKey
```

Comment:
Looks like this one is actually still used by hostgroup tests, I guess I missed it. Perhaps the test could be modified to use factorybot instead of the fixture?

Reply:
:+1: I would prefer to use `FactoryBot` if the record is just for few tests.

Reply:
I'm getting this error when I replaced the fixture to factory bot: 
RuntimeError: stubbed models are not allowed to access the database - LookupValue#save!()

do you think I should also change the tests or I would leave the fixtures?

Reply:
Looking at the test you'd need to have a more complex setup to get it working, needing to associate lookup value with lookup key with puppetclass to the hostgroup. Let's just keep the four fixture for now.